content/docs/command-reference/exp/list.md

```diff
@@ -20,12 +20,12 @@ or commit they're based on. This is similar to `dvc exp show --no-pager`, but
 limited to experiment names and with very simple formatting. See also
 `dvc exp run`.
 
-If a working `git_remote` name (e.g. `origin`) or valid Git repo's URL is
-provided, lists experiments in that <abbr>repository</abbr> instead (if any,
-based on the `dvc remote default`).
+If a working `git_remote` name (e.g. `origin`) or valid Git URL is provided,
+lists experiments in that <abbr>repository</abbr> instead (if any, based on the
```

Comment:
```suggestion
If a working `git_remote` name (e.g. `origin`) or valid Git URL is provided (only SSH authentication is supported),
this command lists the experiments in that <abbr>repository</abbr> instead (if any, based on the
```

Reply:
Here, also. We can use cloned bare repositories on local disk to `dvc exp push`. It's not about particular connection protocol.